### PR TITLE
Fix Accessibility

### PR DESCRIPTION
**Pull request by tasnimchy96**
In this branch we:

- Fix accessibility
      1. Page titles - Fix page title - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L7:~:text=%3Ctitle%3EMy%20Portfolio%3C/title%3E
      2. Image text alternatives - Fix some meaningful short brief to alternative of images.
          - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L15:~:text=%3Cimg%20src,of%20toolbar%22%20/%3E
          - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=Skills-,%3Cimg%20src%3D%22./images/icons/right%2Darrow.svg%22%20alt%3D%22right%2Darrow%22%20/%3E,-%3Cimg%20src
          - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=Frameworks-,%3Cimg%20src%3D%22./images/icons/right%2Darrow.svg%22%20alt%3D%22right%2Darrow%22%20/%3E,-%3Cimg%20src
          - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=%3Cimg%20src%3D%22./images/language%2Dicons/css.svg%22%20alt%3D%22%22%20/%3E
          - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=%3Cimg%20src%3D%22./images/language%2Dicons/html.svg%22%20alt%3D%22%22%20/%3E
          - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=%3Cimg%20src%3D%22./images/language%2Dicons/javascript.svg%22%20alt%3D%22%22%20/%3E
          - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=%3Cimg%20src%3D%22./images/icons/down%2Darrow.svg%22%20alt%3D%22down%2Darrow%22%20/%3E
          - 
      4. Text headings - No issues found
      5. Color contrast - No issues found
      6. Resize - Fix some padding of headline and contact-me section
      7. Interaction - No issues found
      8. Moving content - No issues found
      9. Multimedia - No issues found
      10. The basic structure of the page - Padding changed
            - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=padding%3A%20219px%20360,px%200%20181px%3B
            - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=padding%3A%20142px%20435,250px%20127px%3B
            - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=header%20%7B-,padding%2Dleft%3A%2050px%3B,-width%3A%20100
            - https://github.com/tasnimchy96/Portfolio/pull/7/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL425-L426:~:text=padding%3A%20150,181px%3B

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,17 +4,17 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
+    <title>Tasnim's Portfolio</title>
```
Comment (mckent05) [inline index.html:7]: - Good job done so far, according to the project requirements, `if issues were found,  please link to the corresponding line of code in the Pull Request changes`. Please implement these for all changes made 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @tasnimchy96, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your website is accessible and has no errors, well-done! 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -4,17 +4,17 @@
...
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <!-- Toolbar-Headline Section Starts -->
 
     <div class="white-background">
       <header>
-        <a class="logo">my logo</a>
```
Comment (tasnimchy96) [inline index.html:15]: Changed the logo name

**Review by huseyinbiyik — APPROVED** [review]
Hi team,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
⚠️ Kindly make sure you don't have any uncommented code in your codebase. Please check line 303 on CSS. In this way, you can keep your codebase clean and professional 👍 

-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**